test/flatten.js

```diff
@@ -50,6 +50,31 @@ describe('flatten', function () {
         });
       });
     });
+    describe('null', function () {
+      before(function () {
+        this.obj = {
+          num: new Number(1), // ignore
```

Comment:
why is this ignored by keypather? I don't see that anywhere else, so I'm curious... I see `wiz: null` carries through...